tests/frameworks/layers/l2l4_network_creation_test.py

```diff
@@ -193,6 +193,52 @@ def testL4L2ColumnCreate(self):
     net.run(3)
 
 
+  def testL4L2ColumnLinks(self):
+    """
+    In this simplistic test we create a network and ensure that it has the
+    correct links between regions.
+    """
+
+    # Create a simple network to check its architecture
+    net = createNetwork(networkConfig1)
+
+    links = net.getLinks()
+
+    # Make sure that we have the right number before going on to specifics
+    self.assertEqual(len(list(net.getLinks())), 6, "Incorrect number of links")
+
+    # These are all the links we're hoping to find
+    desired_links=[("sensorInput_0", "dataOut", "L4Column_0", "activeColumns"),
```

Comment:
You're correct that we're very dependent on the region names.  Unfortunately, so far as I can tell, names like L4Column_0 _are_ the real names of the regions, at least so far as the system stores.  As a result, our other tests are equally dependent on this naming scheme. 

I'm somewhat concerned about just printing the lists, since in the more complicated version there are roughly 50 links.  Moreover, the number of links will be noticeably higher if this test scheme is extended for networks with 2D topology.